relay-server/src/actors/project.rs

```diff
@@ -17,7 +17,7 @@ use relay_filter::{matches_any_origin, FiltersConfig};
 use relay_general::pii::{DataScrubbingConfig, PiiConfig};
 use relay_general::store::{BreakdownsConfig, MeasurementsConfig};
 use relay_general::types::SpanAttribute;
-use relay_metrics::{self, Aggregator, Bucket, InsertMetrics, MergeBuckets, Metric};
+use relay_metrics::{self, Bucket, InsertMetrics, MergeBuckets, Metric};
```

Comment:
While we're at it, this `self` can be removed.

```suggestion
use relay_metrics::{Bucket, InsertMetrics, MergeBuckets, Metric};
```